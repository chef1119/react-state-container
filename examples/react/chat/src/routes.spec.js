import React from 'react';
import { shallow } from 'enzyme';

import Routes from './routes';

describe('Test route wrapper unit', () => {
  it('Should match with snapshot', () => {
    const wrapper = shallow(<Routes />);

    expect(wrapper).toMatchSnapshot();
  });
});
