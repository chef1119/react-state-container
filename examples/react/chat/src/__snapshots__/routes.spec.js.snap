// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test route wrapper unit Should match with snapshot 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/chat"
    />
  </Switch>
</BrowserRouter>
`;
